src/Mod/Sketcher/App/ConstraintPyImp.cpp

```diff
@@ -154,8 +154,8 @@ int ConstraintPy::PyInit(PyObject* args, PyObject* /*kwd*/)
             }
 
             if (valid) {
-                constraint->First = FirstIndex;
-                constraint->Second = SecondIndex;
+                constraint->setGeoId(0, FirstIndex);
+                constraint->setGeoId(1, SecondIndex);

```

Comment:
Could it be a good idea to have `First`, `Second`, `Third` getters and setters? It will keep the historical code quite unchanged and is shorter than repeating the element id.

Reply:
we can definitely have special setters / getters for First Second and Third.

Reply:
My suggestion in https://github.com/FreeCAD/FreeCAD/pull/22043#issuecomment-2982951052 (the second one) is exactly that, except it return a reference instead of using a getter/setter.